src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -147,26 +203,25 @@ public EditPersonDescriptor() {}
          * A defensive copy of {@code tags} is used internally.
          */
         public EditPersonDescriptor(EditPersonDescriptor toCopy) {
-            setName(toCopy.name);
```

Comment:
The `createEditedPerson` method suggests that we would be able to edit our `nric` and `name`, but the constructor seems to remove the defensive copies entirely from the creation, even though the `name` and `nric` fields persist. Are we preventing the editing of these fields?

Reply:
Yep.  I don't think we should allow the editing of names and nric.